### DISCUSSION TITLE
How do I find a .gba file after clicking open

### DISCUSSION THREAD

**Discussion by IMDAHBIGBOSS** [visualboyadvance-m/visualboyadvance-m#1165] [Q&A]
Currently I'm looking for my file called GamerGame.gba but I can't find it. Note Im using the linux edition and im on a chromebook.![image](https://github.com/visualboyadvance-m/visualboyadvance-m/assets/141575912/e129d383-0970-46c9-a260-6aa463af582d)

What do I do?

**Comment by ZachBacon** [comment]
Look where you downloaded it.
